.gitignore

```diff
@@ -28,5 +28,3 @@ node_modules
 
 .DS_Store
 *.iml
-
-dist
```

Comment:
Are we removing this to make it readily available to be used in the browser instead of having to run the npm command to generate?